easybuild/easyconfigs/a/AlphaPulldown/AlphaPulldown-0.30.4-foss-2020b.eb

```diff
@@ -0,0 +1,93 @@
+# created by Denis Kristak (Inuits)
+easyblock = "PythonBundle"
+
+name = "AlphaPulldown"
+version = "0.30.4"
+
+homepage = "https://github.com/KosinskiLab/AlphaPulldown"
+description = """AlphaPulldown is a Python package that streamlines protein-protein
+interaction screens and high-throughput modelling of higher-order oligomers using AlphaFold-Multimer"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+builddependencies = [
+    # required for installing dm-tree
+    ('Bazel', '3.7.2'),
```

Comment:
Can be removed if we use a proper `dm-tree` dependency

Reply:
resolved